tests/ext/aiohttp/test_middleware.py

```diff
@@ -5,7 +5,10 @@
 """
 import asyncio
 from aws_xray_sdk import global_sdk_config
-from unittest.mock import patch
+try:
+    from unittest.mock import patch
+except ImportError:
+    from mock import patch  # Python 2
```

Comment:
I like putting the comments above but this is just an opinion. It's fine the way it is too.

```suggestion
except ImportError:
    # NOTE: Python 2 dependency
    from mock import patch
```

Reply:
Adjusted in the three files where this sort of inline comment appears.